cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -48,7 +48,7 @@ def test_myservice_crud(appliance, setup_provider, context, order_service):
         my_service.set_ownership("Administrator", "EvmGroup-approver")
         my_service.update({'description': '{}_edited'.format(catalog_item.name)})
         if appliance.version > "5.8":
-            my_service.edit_tags("Cost Center", "Cost Center 001")
+            my_service.edit_tags("Cost Center", "Cost Center 002")
```

Comment:
If possible remove above 5.8 condition

Reply:
@niyazRedhat bringing discussion about BZ impact here so we can have thread and not just PR comments.

Given: 1) don't want to block test on BZ, 2) does not really matter which cost center we pick for this test

I say we:
1) take @digitronik suggestion and remove the 5.8 block that this is in
2) add in-line comment about BZ impacting tag visibility
3) use wt.core.util.partial_match here, passing `partial_match('Cost Center')` to just pick the first visible cost center tag.  `edit_tags` method just passes this value to a `fill()` call, so this should work fine.

Reply:
@mshriver I tried to add partial_match but it is not working on ssui edit tag form.
Bug will be fixed in a month. Now can we go for  ```Cost Center 002``` ?

Traceback
```ipdb> view.form.tag_name.fill(partial_match("Cost Center"))
*** ValueError: Cannot locate option with visible text: u'Cost Center'. Available options: u'Select a value to assign', u'Cost Center 002'
```

Reply:
I would like to address the failure of partial_match in this case. Do you have a python notebook you can share, or even just a set of miq shell commands to reproduce? The snippet you included there clearly includes 'Cost Center', so it should have matched.